policy/modules/apps/irc.te

```diff
@@ -112,6 +128,10 @@ userdom_user_content_access_template(irc, irc_t)
 
 xdg_manage_downloads(irc_t)
 
+tunable_policy(`irc_allow_screen_access',`
```

Comment:
Regardless if we keep the tunables, this should be in an optional block, otherwise it will always need screen.